src/applications/mhv/medical-records/tests/e2e/medical-records-download-txt-allergies.cypress.spec.js

```diff
@@ -12,8 +12,7 @@ describe('Medical Records View Allergies', () => {
     cy.visit('my-health/medical-records');
     cy.reload({ force: true });
     // Given Navigate to Allergy Page
-    cy.intercept('POST', '/my_health/v1/medical_records/session').as('session');
-    cy.wait('@session');
+
```

Comment:
I just want to double-check that you cleared some of these e2e changes with QA. This one in particular.

Reply:
I didn't do this.